src/pages/home/home.ts

```diff
@@ -11,6 +11,7 @@ export class HomePage {
   karma: string
   hamcode: string
   lunchers: any
+  lunchersAmount: any
```

Comment:
The class property 'lunchersAmount' must be marked either 'private', 'public', or 'protected'<br>Missing semicolon